packages/coinjoin/src/client/CoinjoinRound.ts

```diff
@@ -370,8 +370,8 @@ export class CoinjoinRound extends EventEmitter {
         // set error on each input
         spentInputs.forEach(input => {
             input.clearConfirmationInterval();
-            input.setError(new Error(WabiSabiProtocolErrorCode.InputSpent));
-        }); // TODO: error same as wasabi coordinator?
+            input.setError(new Error('InputSpent'));
```

Comment:
This `Error('InputSpent')` is something different than `WabisabiProtocolException` with `WabiSabiProtocolErrorCode.InputSpent`?

Reply:
i forgot about this one :)
yes, this is something different. this is just regular error assigned to registered inputs after account update.
since it's not a response from coordinator then `WabisabiProtocolException` will never be parsed/cheked.

input with assigned error will be detained after processing round.

i've changed error message in fixup below to avoid confusion